aten/src/ATen/native/cuda/UnaryGeometricKernels.cu

```diff
@@ -92,9 +92,30 @@ void sinh_kernel_cuda(TensorIterator& iter) {
   });
 }
 
+// We manually overload sinh because std::sinh does not work with thrust::complex types.
```

Comment:
copy-pasted comment, please change sinh to tanh

Reply:
Oops. Will do that. Thanks.